src/runtime/virtcontainers/qemu.go

```diff
@@ -2101,7 +2101,7 @@ func genericAppendBridges(devices []govmmQemu.Device, bridges []types.Bridge, ma
 				ID:   b.ID,
 				// Each bridge is required to be assigned a unique chassis id > 0
 				Chassis: idx + 1,
-				SHPC:    true,
+				SHPC:    false,
```

Comment:
If Kata no longer uses this, and Kata is the only consumer of govmm, can't we zap this element from `govmm`'s `BridgeDevice struct` now?

Reply:
I don't think we're the only consumer - I'd err on the side of keeping functionality.

Reply:
It's become apparent we're not.  That's an ongoing discussion elsewhere, but it's not really in scope for this patch.